fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -389,7 +389,13 @@ public interface Count extends StoreTimer.Event {
 
     /**
      * {@link Event}s that count the number of occurrences of an operation and also the size of the operation.
-     * There is no meaningful time duration associated with these events.
+     * {@link SizeEvent} is similar in implementation to (Time){@link Event} except for the fact that the recorded value
+     * is not the time. It is also different from {@link Count} since along with the size of the event in concern, it
+     * also records the number of times the record has occurred.
```

Comment:
```suggestion
     * also records the number of times the event has occurred.
```

In theory, you could re-word this as "the number of times the event has been recorded", I think.